### DISCUSSION TITLE
Why do you keep renaming the repos?

### DISCUSSION THREAD

**Discussion by BrandonHicks-msr** [ixti/sidekiq-throttled#137] [General]
I really appreciate the work you have done on this gem, but why do you keep renaming the repos?

### **For context:**
Yesterday AM it seems like you took over the repo from sensortower and renamed your previous repo to `sidekiq-throttled-new`, and pulled in their previous commits. This afternoon you switched your repo this from `sidekiq-throttled-new` to `sidekiq-throttled-clone` and synced `sidekiq-throttled-clone` into `sidekiq-throttled`. This has left people who use these gems that reference these repos in disarray. ie: `gem 'sidekiq-throttled', github: 'ixtix/sidekiq-throttled'`

**Comment by OskarEichler** [comment]
Looks like @ixti is doing some house-keeping, and from what I see everything is setup properly again now!

**Reply by ixti** [reply]
Sorry about this. Previous (original) repo was transferred to me but the process was a bit far from smooth. Everything should be stble from now on.

**Comment by ixti** [comment]
Had issues with getting back control over the original repo. Should be stable now.

**Comment by ixti** [comment]
sidekiq-throttled-clone will soon to be gone, once I'll get my hands on sidekiq-pro compatibility PR
